src/codegeneration/module/ExportListBuilder.js

```diff
@@ -25,9 +24,7 @@ import {transformOptions} from '../../Options.js';
  * @param {Loader} loader
  * @return {void}
  */
-export function buildExportList(deps, loader, reporter) {
-  if (!transformOptions.modules)
```

Comment:
Because the `modules` option can never be falsy.  Scripts don't have deps. I just added a check to ensure we never enter this function for scripts.